### DISCUSSION TITLE
Unable to activate hypervisor on M1 iPad

### DISCUSSION THREAD

**Discussion by Dr-TSNG** [utmapp/UTM#6111] [UTM Help (iOS)] [answered]
**Describe the issue**
A clear and concise description of what the issue is. **BEFORE SUBMITTING YOUR ISSUE, PLEASE LOOK AT THE PINNED ISSUES AND USE THE SEARCH FUNCTION TO MAKE SURE IT IS NOT ALREADY REPORTED. ALWAYS COMMENT ON AN EXISTING ISSUE INSTEAD OF MAKING A NEW ONE.**

I installed UTM-HV.ipa in TrollStore, but the virtualization button is gray and I can't use it. I tried to install the deb, but unfortunately dpkg says `package architecture (iphoneos-arm) does not match system (iphoneos-arm64)`.

**Configuration (required)**
* UTM Version: 4.4.5
* OS Version: 16.4
* Device Model: iPad Air M1
* Is it jailbroken (name jailbreak used)? Yes, Dopamine
* How did you install UTM? TrollStore

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

Not available since virtualization button is gray.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by osy** [comment, marked as answer]
https://twitter.com/utmapp/status/1708907045314035986
